dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task05380ChangeContainerPathToAbsolute.java

```diff
@@ -99,15 +101,24 @@ private String replaceAllContainerRelativePath(
     }
 
     private List<String> getRelativePaths(final String drawedBody) {
-        final TemplateLayout templateLayoutFromJSON = DotTemplateTool.getTemplateLayout(drawedBody);
+        final Set<ContainerUUID> templateContainers = DotTemplateTool.getTemplateContainers(
```

Comment:
Let's use only private methods of this UT. If you need to copy paste code into this UT and make then private methods go ahead.

Reply:
done https://github.com/dotCMS/core/pull/22604/commits/e0ffb54258b3a929d89c7089ffca177a18823ea0#diff-36b521520ed8ced8db845910ef888502f0dbedb7c83cc53b7bb39cb9d7de60d2L237

done https://github.com/dotCMS/core/pull/22604/commits/e0ffb54258b3a929d89c7089ffca177a18823ea0#diff-b8921f877ae173e9ff875d93630f0f026b4b96f3d23e0a98d3b768208be93399R137